braindecode/models/shallow_fbcsp.py

```diff
@@ -194,21 +205,23 @@ def __init__(
         if self.batch_norm:
             init.constant_(self.bnorm.weight, 1)
             init.constant_(self.bnorm.bias, 0)
-        init.xavier_uniform_(self.conv_classifier.weight, gain=1)
-        init.constant_(self.conv_classifier.bias, 0)
+        init.xavier_uniform_(self.final_layer[0].weight, gain=1)
```

Comment:
and here I would also use directly the name of the module - `conv_classifier`, just in case someone adds one more layer in the beginning  of `final_layer` then the code won't crash